jobs/component_chart_publish.groovy

```diff
@@ -86,11 +86,16 @@ repos.each { Map repo ->
               # package release chart
               helm package ${repo.chart}
 
-              # download index file from aws s3 bucket
-              aws s3 cp s3://helm-charts/${repo.chart}/index.yaml .
-
-              # update index file
-              helm repo index . --url https://charts.deis.com/${repo.chart} --merge ./index.yaml
+              if [ "$(git tag | wc -l)" == "1" ]; then
```

Comment:
we need to escape the `$` here (`\$`) b/c it is a shell/runtime resolution as opposed to a groovy/job build-time resolution...